### ISSUE TITLE
Actcritic not converging in the end

### ISSUE THREAD

**Issue by Hanzhang-lang** [DigZator/Gym-Policy-Gradient#4]
I've run the script code provided in the actcri , after running 1500 episodes,  running test(10) i got rewards still around -1200, the pic is shown below, was i doing wrong?
<img width="478" alt="image" src="https://user-images.githubusercontent.com/66400564/167258528-c301144e-f90f-44ef-90df-86da26d8e355.png">

**Comment by DigZator** [comment]
No, its not wrong. The algorithm is not able to converge, if you know how to fix this error please let me know.
